docs/conversion_examples_gallery/recording/abf.rst

```diff
@@ -0,0 +1,143 @@
+ABF data conversion
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Convert ABF intracellular electrophysiology data to NWB using :py:class:`~neuroconv.datainterfaces.icephys.abf.abfdatainterface.AbfInterface`.
+
+.. code-block:: python
+
+    >>> from neuroconv import NWBConverter, AbfInterface
+
+    >>> # Define the Data Interfaces for your converter
+    >>> class MyIcephysConverter(NWBConverter):
+    >>>     data_interface_classes = dict(AbfInterface=AbfInterface)
```

Comment:
Could you make this work without an `NWBConverter`, just using the AbfInterface, like the other conversion examples?

Reply:
done